.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-assign", "transform-object-rest-spread"],
+  "presets": ["env", "react"],
```

Comment:
Why the update to 'env'

Reply:
![screen shot 2018-02-23 at 12 32 46 pm](https://user-images.githubusercontent.com/34115417/36610375-ae3ca67c-1895-11e8-9e27-84b470f17fe1.png)